lib/include/ultrahdr/editorhelper.h

```diff
@@ -19,6 +19,7 @@
 
 #include "ultrahdr_api.h"
 #include "ultrahdr/ultrahdrcommon.h"
+#include "ultrahdr/jpegr.h"
```

Comment:
I think this is not needed. Could you double check?

Reply:
This is needed since we are making use of isBufferDataContiguous function declared in jpegr.h

Reply:
@vivekrj1806  as editorhelper.h functionality is mostly isolated, it does not make sense for it to include jpegr.h, If you move is isBufferDataContiguous inside ultrahdrcommon.h then there is no need to include jpegr.h

Reply:
Done.